### PR TITLE
[DPEDE-834] Added column customization to Modal docs

### PR DESCRIPTION
**Pull request by wojciechwlodarczyklumen**
https://ctl.atlassian.net/browse/DPEDE-834

### CODE REVIEW

File: src/website/views/components/modal/_examples.pug [hunk 1]
```diff
@@ -700,12 +703,151 @@ p.-text
...
+    },
+    data: {
+      columnsData: {
+        columns: exampleColumns,
+      },
+    },
+    mounted() {
+      if (window.Cypress) {
+        window.baseColumnCustomizationExample = this;
+      }
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/modal/_examples.pug:851]: Why do we need this as it relates to the e2e tests?

File: src/website/views/components/modal/_examples.pug [hunk 2]
```diff
@@ -700,12 +703,151 @@ p.-text
...
+      li.-active
+        a(href=`#vue-no-results`) Vue
+      li.-disabled
+        a(href=`#html-no-results`) HTML Blueprint
+  .chi-tabs-panel#webcomponent-no-results
+  .chi-tabs-panel.-active#vue-no-results
+    .chi-tab__description Use <code>columns-data</code> to customize the columns.
+    :code(lang='html')
+      <!-- Vue component -->
+      <ChiColumnCustomization id="column-customization" ref="baseColumnCustomizationRef" :columns-data="columnsData"></ChiColumnCustomization>
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/modal/_examples.pug:724]:
```suggestion
      <ChiColumnCustomization id="column-customization" ref="baseColumnCustomizationRef" :columnsData="columnsData"></ChiColumnCustomization>
```

File: src/website/views/components/modal/_examples.pug [hunk 3]
```diff
@@ -700,12 +703,151 @@ p.-text
         <!-- JavaScript -->
         <script>chi.modal(document.getElementById('modal-trigger-9'));</script>
 
+h3 Column Customization
+p.-text
+  | Opens column customization modal, and allows to modify them.
+.example.-mb--3
+  chi-vue-column-customization(id="column-customization" ref="baseColumnCustomizationRef" :columns-data="columnsData")
+  .example-tabs.-pl--2
+    ul.chi-tabs#example-no-results
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/modal/_examples.pug:712]: Take care with the IDs

```suggestion
    ul.chi-tabs#example-column-customization-modal
```

File: src/website/views/components/modal/_examples.pug [hunk 4]
```diff
@@ -700,12 +703,151 @@ p.-text
...
+      li.-disabled
+        a(href=`#webcomponent-no-results`) Web Component
+      li.-active
+        a(href=`#vue-no-results`) Vue
+      li.-disabled
+        a(href=`#html-no-results`) HTML Blueprint
+  .chi-tabs-panel#webcomponent-no-results
+  .chi-tabs-panel.-active#vue-no-results
+    .chi-tab__description Use <code>columns-data</code> to customize the columns.
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/modal/_examples.pug:721]:
```suggestion
    .chi-tab__description Use <code>columnsData</code> to customize the columns.
```

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

File: src/website/views/components/modal/_examples.pug [hunk 5]
```diff
@@ -700,12 +700,183 @@ p.-text
...
+      li.-active
+        a(href=`#vue-column-customization-modal`) Vue
+      li.-disabled
+        a(href=`#html-column-customization-modal`) HTML Blueprint
+  .chi-tabs-panel#webcomponent-column-customization-modal
+  .chi-tabs-panel.-active#vue-column-customization-modal
+    .chi-tab__description Use <code>columnsData</code> to customize the columns.
+    :code(lang='html')
+      <!-- Vue component -->
+      <ChiColumnCustomization id="column-customization" ref="baseColumnCustomizationRef" :columnsData="columnsData"></ChiColumnCustomization>
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/modal/_examples.pug:773]:
```suggestion
      <ChiColumnCustomization :columnsData="columnsData"></ChiColumnCustomization>
```

**Review by josecarlosgonzalezv — APPROVED** [review]

**Review by jllr — APPROVED** [review]

**Pull request merged**